lib/src/components/comms_button/comms_button.dart

```diff
@@ -0,0 +1,426 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../../zeta_flutter.dart';
+
+/// Enum for the type of comms button.
+enum ZetaCommsButtonType {
+  /// Green background, no border, white icon
+  positive,
+
+  /// Red background, no border, white icon
+  negative,
+
+  /// Light grey background, dark grey border, black icon
+  on,
+
+  /// Dark grey background, light grey border, white icon
+  off,
+
+  /// Transparent background, red border, red icon
+  warning,
+}
+
+/// Comms button component.
+/// This component is used to display a button for communication action. Answer, reject, mute, hold, speaker, etc.
```

Comment:
Probably worth putting in something here about using the named constructors to create specific icons